modules/virt-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -6,16 +6,19 @@
 [id="virt-troubleshooting-incorrect-policy-config_{context}"]
 = Troubleshooting an incorrect node network configuration policy configuration
 
-You can apply changes to the node network configuration across your entire cluster by applying a node network configuration policy.
-If you apply an incorrect configuration, you can use the following example to troubleshoot and correct the failed node network policy.
+You can apply changes to the node network configuration across your entire cluster by applying a node network configuration policy. 
 
-In this example, a Linux bridge policy is applied to an example cluster that has three control plane nodes and three compute nodes.
-The policy fails to be applied because it references an incorrect interface.
-To find the error, investigate the available NMState resources. You can then update the policy with the correct configuration.
+If you applied a wrong configuration, you can use the following example to troubleshoot and correct the failed node network policy. The example attempts to apply a Linux bridge policy to a cluster that has three control plane nodes and three compute nodes. The policy is not applied because the policy references the wrong interface. 
+
+To find an error, you need to investigate the available NMState resources. You can then update the policy with the correct configuration.
+
+.Prerequisites
+
+* You ensured that an `ens01` interface does not exist on your Linux system.
 
 .Procedure
 
-. Create a policy and apply it to your cluster. The following example creates a simple bridge on the `ens01` interface:
+. Create a policy your cluster. The following example creates a simple bridge, `ens01`, on the `br1` interface:
```

Comment:
"creates a simple bridge, ens01, on the br1 interface" is incorrect; you want something like "creates a simple bridge, br1, containing esn01 as its member"

Reply:
Thanks, Mat. Updated!